### PR TITLE
add internationalization for existing components

### PR DESCRIPTION
**Pull request by DimitarShemet**
Add internationalization

### CODE REVIEW

File: src/components/translations/en.js [hunk 1]
```diff
@@ -1,8 +1,40 @@
-export let messages_en= {
```
Comment (Terapool) [inline src/components/translations/en.js:1]: Please, move _translations_ folder one level up (replace from '_components_' to '_app_' folder).

File: src/components/translations/en.js [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
-    "app.name": "Lessoner",
-    "app.categories": "Categories",
-    "app.lessons": "Lessons",
-    "app.about": "About",
-    "app.login": "LogIn",
-    "app.registration": "Register",
+import { minSymbol } from "../FirstRegistrationForm"
+import { maxSymbol } from "../FirstRegistrationForm"
+
+export let messages_en = {
```
[the remark this answers is not in the document]
Comment (DimitarShemet) [reply src/components/translations/en.js:4]: Yes, for example in the future, we wil get new data about this?

File: src/components/translations/ru.js [hunk 3]
```diff
@@ -1,8 +1,40 @@
...
+"app.genderSelector.gender": "Выберите свой пол",
+"app.passwordAndConfirm.pass": "Пароль",
+"app.passwordAndConfirm.confirmPass": "Подтвердите пароль",
+"app.phoneNumberPage.title": "Введите номер вашего телефона",
+"app.phoneNumber.label": "Номер телефона",
+"app.phoneNumberPage.submit":"Отправить код",
+"app.loginPage.title": "Зарегистрироваться на Учитель",
+"app.loginPage.checkbox": "Оставаться в системе",
+"app.loginPage.password": "Забыли пароль?",
+"hello": "Привет, мир",
```
[the remark this answers is not in the document]
Comment (DimitarShemet) [reply src/components/translations/ru.js:39]: Hello, everyone:)

File: src/translations/en.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+"app.firstRegistrationForm.button": "Next",
+"app.genderSelector.gender": "Select Your Gender",
+"app.passwordAndConfirm.pass": "Password",
+"app.passwordAndConfirm.confirmPass": "Confirm password",
+"app.phoneNumberPage.title": "Enter your phone number",
+"app.phoneNumberPage.submit":" Submit code",
+"app.phoneNumber.label": "Phone number",
+"app.loginPage.title": "Login to the Lessoner",
+"app.loginPage.checkbox": "Stay logged in",
+"app.loginPage.password": " Forgot your password?",
```
Comment (Terapool) [inline src/translations/en.js:38]: Remove a space before 'Forgot'

File: src/translations/ru.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+"app.firstRegistrationForm.passwordLength": `Пароль должен быть от  ${minSymbol} до ${maxSymbol} символов`,
+"app.firstRegistrationForm.passwordConfrim": "Пароли не совпадают",
+"app.firstRegistrationForm.termsAndConditions": "Вы должны дать согласие на обработку ваших персональных данных в соответствии с Условиями",
+"app.firstRegistrationForm.button": "Дальше",
+"app.genderSelector.gender": "Укажите свой пол",
+"app.passwordAndConfirm.pass": "Пароль",
+"app.passwordAndConfirm.confirmPass": "Подтвердите пароль",
+"app.phoneNumberPage.title": "Введите номер вашего телефона",
+"app.phoneNumber.label": "Номер телефона",
+"app.phoneNumberPage.submit":"Отправить код",
```
Comment (Terapool) [inline src/translations/ru.js:33]: add space after ":"

File: src/translations/en.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+"app.lessoner": "Lessoner",
+"app.sign_in": "Sign_in",
+"app.sign_up": "Sign_up",
+"app.birthdaylabel": "When is your birthday?",
+"app.checkbox": "I agree to the processing of my personal data in accordance with the",
+"app.checkbox.terms": "Terms",
+"app.email.name": "Email",
+"app.firstRegistrationForm.existsInDb": "This email address is already registered",
+"app.firstRegistrationForm.invalidationRules": "Please enter a valid email address",
+"app.firstRegistrationForm.passwordRegEx": `An invalid character is present in the password. Password must be between ${minSymbol} and ${maxSymbol} characters; upper or lower case Latin letters (a–z, A–Z); numbers from 0 to 9; symbols ! # $ % & ' * + - / = ? ^ _  { | } ~`,
```
Comment (Terapool) [inline src/translations/en.js:25]: Use templete here without import of constants. **Example:**

"app.form.message": 'Your password must be {minValue} - {maxValue}'

then when using:

intl.formatMessage({ id: 'app.form.message', {minValue: 1, maxValue: 5 }});

https://formatjs.io/docs/intl/#message-formatting-fallbacks

File: src/translations/en.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+"app.firstRegistrationForm.passwordLength": `Password must be between ${minSymbol} and ${maxSymbol} characters`,
+"app.firstRegistrationForm.passwordConfrim": "Passwords do not match",
+"app.firstRegistrationForm.termsAndConditions": "You must consent to the processing of your personal data, in accordance with the Terms",
+"app.firstRegistrationForm.button": "Next",
+"app.genderSelector.gender": "Select Your Gender",
+"app.passwordAndConfirm.pass": "Password",
+"app.passwordAndConfirm.confirmPass": "Confirm password",
+"app.phoneNumberPage.title": "Enter your phone number",
+"app.phoneNumberPage.submit":" Submit code",
```
Comment (Terapool) [inline src/translations/en.js:34]: Please, fix space issues: 

"app.phoneNumberPage.submit":" Submit code", -> "app.phoneNumberPage.submit": "Submit code",

File: src/translations/en.js (hunk 6 above)
Comment (Evgeniy121212) [reply src/translations/en.js:25]: +

Comment (Evgeniy121212) [reply src/translations/en.js:25]: agreed

**Review by Evgeniy121212 — APPROVED** [review]
approving ahead

**Review by Terapool — APPROVED** [review]

**Review by IgorTanov — APPROVED** [review]

File: src/components/Email.tsx [hunk 8]
```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import './Email.scss';
 import { EMAIL } from "../constants";
+import { FormattedMessage } from "react-intl";
+
```
Comment (Kurilo-Pavel) [inline src/components/Email.tsx:5]: delete empty line

File: src/components/FirstRegistrationForm.tsx [hunk 9]
```diff
@@ -28,10 +30,12 @@ const emailInvalidationRules = [
 ];
 
 const minSymbol = 6;
-const maxSymbol = 256;
+const maxSymbol=256;
```
Comment (Kurilo-Pavel) [inline src/components/FirstRegistrationForm.tsx:33]: add space for "maxSymbol = 256;"

**Pull request merged**